source/hardware/controllers/behringer_ddm4000.rst

```diff
@@ -35,43 +43,104 @@ Configure at least 1 mixer section as MIDI controller:
+No.  Control                                                  Function
+===  =======================================================  ============================================================================================
+4    :hwlabel:`TALK ON`                                       Switch the Talkover function.
+===  =======================================================  ============================================================================================
+
+
+.. _behringer-ddm4000-crossfader:
+
+Crossfader section (p. 7)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/behringer_ddm4000_page7_crossfader.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Behringer DDM4000 (crossfader section)
+   :figclass: pretty-figures
+
+   Crossfader section
+
+===  =======================================================  ============================================================================================
+No.  Control                                                  Function
+===  =======================================================  ============================================================================================
+20   Crossfader                                               Fade between the signals that are assigned to its two sides A and B.
+21   :hwlabel:`CF ON` button                                  Toggle crossfader function. When disabled, the crossfader control in the Mixxx user interface is hidden.
+24   :hwlabel:`CURVE` knob                                    Adjust the response of the crossfader.
+25   :hwlabel:`REVERSE HOLD` button                           Toggle a permanent reverse of the crossfader sides A and B. This means that A and B are interchanged.
+26   :hwlabel:`REVERSE TAP` button                            Toggle a momentary reverse of the crossfader sides A and B. This means that A and B are interchanged as long as the TAP push button is held down.
+===  =======================================================  ============================================================================================
+
+
+.. _behringer-ddm4000-sampler:
+
+Sampler (p. 8)
+~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/behringer_ddm4000_page8_sampler.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Behringer DDM4000 (sampler)
+   :figclass: pretty-figures
+
+   Sampler section
+
+===  =======================================================  ============================================================================================
+No.  Control                                                  Function
+===  =======================================================  ============================================================================================
+55   :hwlabel:`VOLUME/MIX` knob                               Control the volume of sampler 1.
```

Comment:
```suggestion
54   :hwlabel:`VOLUME/MIX` knob                               Control the volume of sampler 1.
```

Reply:
Actually, the image has a typo (in the description of the manual, the knob is referenced as no. 55, but on the image, no. 54 is listed twice and no. 55 is missing). I fixed the image.